_posts/2019-09-19-serverless-adventures-in-a-new-dimension.markdown

```diff
@@ -0,0 +1,70 @@
+
+## Into a New Dimension
+
+The history of software development & delivery has a number of step changes which unlocked new business potential. This is usually coupled with the addition of a ‘new dimension’ in thinking.
+
+Imaging you are living on a 2-D plane. You are walking along your 2D road one day and you come across an obstacle. Let’s call it a line. The next day you are walking down the same road and you come across the same thing. Another pesky line. But is it really?
+
+![Life on a 2D plane](/img/serverless/2d.png "Life on a 2D plane")
+
+To the rest of us living in a 3rd dimension, we can see that it’s not a line, it’s a box. But to you, without the perspective of the other dimension, it just looks the same as the thing you saw yesterday.
+
+## Software Step Changes
+
+Why is this important? Let’s take the internet for example. Before the internet, people bought their software from a shop. Software could be written slowly because the distribution chain was slow. But then the internet opened up a new dimension for people to distribute software. A small group of people who recognised the power of this new dimension started writing software in their houses and distributing it online. Suddenly, software could be distributed instantly and cheaply, value could be given to customers faster and customer feedback could even be translated into new features quicker.
+
+![Level Up](/img/serverless/level_up.jpg "Level Up!")
+
+The people who harnessed the power of this new dimension led the way into a new era and the old era pretty much became obsolete. Now most of the software we use everyday is downloaded from an app store.
+
+Let’s look at another more relevant example. We all know that software ultimately has to run on a server somewhere. But Virtual Machines introduced a new dimension to computer hardware. No longer did you have to dedicate one physical machine to an application; suddenly you could run multiple machines on one physical one. The new virtual machines could be created almost instantly at zero cost, and were also disposable. There was no more waiting for hardware to be delivered, and hardware specifications turned into machine configuration. This new dimension brought forth repeatable configurations, cloud computing and containers, which is almost becoming the new norm.
+
+## The Next Compute Dimension
+
+![Next Dimension](/img/serverless/new_dimension.jpg "The Next Dimension")
+
+The next step-change in web-based software delivery is already here, and has been here for a few years. That change is serverless computing, but as with many new things, it is not fully understood.
+
+Why do I refer to it as the next dimension? Because there are many things that it brings that you can’t see without changing your perspective, along with a new way of thinking that is needed to unlock them.
+Total hardware abstraction
```

Comment:
This should be a heading?

`### Total hardware abstraction`

As an element of style, I'd capitalise headings.